src/content/drafts/custom-formatter-blog-post.md

```diff
@@ -0,0 +1,351 @@
+  // console.log(tomlResults);
+});
+```
+
+With everything ready, let's run the test:
+
+```bash
+npm test
+```
+
+In the terminal you should see the following output with the passing test results:
+
+```
+> eslint-formatter-toml@1.0.0 test
+> jest
+
+ PASS  ./formatter.test.js
+  ✓ Test TOML formatter (180 ms)
+
+Test Suites: 1 passed, 1 total
+Tests:       1 passed, 1 total
+Snapshots:   0 total
+Time:        1.04 s, estimated 2 s
+Ran all test suites.
+```
+
+Everything is working as expected!
+
+### 4. Publish to npm
+
+Since we've validated that the custom formatter works the way we expect from the tests,
+we are ready to publish it to npm. Once it's published, anyone on the internet
+who wants to use the custom formatter is able to.
+
+Update the project's `package.json` file with the following information:
+
+| **Field**     | **Value**  |
+|:--------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `"name"`        | A unique string name for the package. The package name **must** begin with `eslint-formatter-`. There cannot be any other packages published on npm with the same name. |
+| `"version"`     | A string version for the package, like `"1.0.0"`. |
+| `"description"` | A string description of the package.|
+| `"main"`        | `"formatter.js"` |
+| `"keywords"`    | `["eslint-formatter", "eslintformatter", "eslint"]`. Add any other keywords you'd like to the array. |
+| `"author"`      | String or object with any author information you'd like to add. For more information, refer to the [npm documention](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#people-fields-author-contributors). |
+
+Your `package.json` should look similar to the following: 
+
+```json
+{
+  "name": "eslint-formatter-toml",
+  "version": "1.0.0",
+  "description": "ESLint custom formatter that publishes the formatter output in TOML",
+  "main": "formatter.js",
+  "keywords": ["eslint-formatter", "eslintformatter", "eslint"],
+  "author": {
+    "name" : "Ben Perlmutter",
+    "email" : "ben@perlmutter.io",
+    "url" : "http://ben.perlmutter.io/"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "json2toml": "^3.0.1"
+  },
+  "devDependencies": {
+    "eslint": "^8.23.0",
+    "jest": "^29.0.1",
+    "toml": "^3.0.0"
+  }
+}
+```
+
+The package is ready for publication. To publish it, run the following command
+from the main directory of the project:
+
+```bash
+npm publish
+```
+
+Now the formatter is live for anyone to use! You can find it by visiting the link
+`https://www.npmjs.com/package/eslint-formatter-<REST-OF-YOUR-PACKAGE-NAME>`.
+
+### 5. Use Published Custom Formatter
+
+Use the published package by installing the custom formatter in your project
+with the following command: 
+
+```bash
+npm install --save-dev eslint-formatter-<REST-OF-YOUR-PACKAGE-NAME>
```

Comment:
Because you spent this post explaining how to create eslint-formatter-toml, I think it would be better to show readers how to use that package directly throughout the rest of this section. So instead of "REST-OF-YOUR-PACKAGE-NAME", go ahead and use "toml".